### ISSUE TITLE
hard triplet loss

### ISSUE THREAD

**Issue by ffredd** [Mu-ING/Person-ReID-Triplet-Loss-Model-based-on-Tensorflow#1]
"Then we could choose postive with highest similairty, if positive exists, and negative with lowest similarity for each anchor"
Is this statement contrary? Shouldn't it be to find the positive pair with the smallest similarity and the negative pair with the largest similarity?

**Comment by ffredd** [comment]
and can u provide ur email?

**Comment by Mu-AIZIP** [comment]
U are right and thanks for your comment! The most similar triplet should be found. U can find me with this email qing@aizip.ai

**Comment by ffredd** [comment]
when I use hard-triplet-loss to train the model, it's difficult to converge~ Even though the loss of my training set is decreasing, the loss of my validation set is almost unchanged. Have you ever had such a problem? How did you solve it?
